sparts/vtask.py

```diff
@@ -201,6 +201,7 @@ def set_exception(self, exception):
         self.timer.stop()
         if self.future is not None:
             self.future.set_exception(exception)
+            handled = True
```

Comment:
Add a comment above, like:
```python
# For future-based work, let's allow the submitter to handle the exception
# gracefully.  Since we can't know if that was successful, just assume the
# exception will be handled (or re-raised) by someone awaiting the future.
# This isn't guaranteed, but is a reasonable assumption.